<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>450</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>450</height>
    </size>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>802</width>
      <height>452</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QFrame" name="left_frame">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>450</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>450</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(27, 29, 35);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>201</width>
          <height>152</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QPushButton" name="libraryTabButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {	
	border: none;
/*	text-align: center;*/
	color: rgb(255,255,255);
	text-align: left;;
	padding-left: 50px;
}
QPushButton:hover {
	background-color: rgb(33, 37, 43);
}
QPushButton:pressed {	
	background-color: rgb(44, 49, 60);
}
QPushButton:checked {	
	background-color: rgb(44, 49, 60);
}</string>
           </property>
           <property name="text">
            <string>Bibliothek</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="classicTabButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {	
	border: none;
/*	text-align: center;*/
	color: rgb(255,255,255);
	text-align: left;;
	padding-left: 50px;
}
QPushButton:hover {
	background-color: rgb(33, 37, 43);
}
QPushButton:pressed {	
	background-color: rgb(44, 49, 60);
}
QPushButton:checked {	
	background-color: rgb(44, 49, 60);
}</string>
           </property>
           <property name="text">
            <string>Mosaic erstellen</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="partialTabButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {	
	border: none;
/*	text-align: center;*/
	color: rgb(255,255,255);
	text-align: left;;
	padding-left: 50px;
}
QPushButton:hover {
	background-color: rgb(33, 37, 43);
}
QPushButton:pressed {	
	background-color: rgb(44, 49, 60);
}
QPushButton:checked {	
	background-color: rgb(44, 49, 60);
}</string>
           </property>
           <property name="text">
            <string>Teil-Mosaic erstellen</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>450</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>450</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(44, 49, 60);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="libraryPage">
        <widget class="QLineEdit" name="libraryLineEdit">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>20</y>
           <width>441</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {	
	border: none;
	color: rgb(255,255,255);
	background-color: rgb(27, 29, 35);
	border-radius: 5;
}</string>
         </property>
         <property name="text">
          <string>D:\Projekt\Uni\Medienverarbeitung\res\out</string>
         </property>
        </widget>
        <widget class="QLabel" name="libraryLabel">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>20</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255,255,255);</string>
         </property>
         <property name="text">
          <string>Bibliothek</string>
         </property>
        </widget>
        <widget class="QPushButton" name="importButton">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>50</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {	
	border: rgb(255,255,255);
	color: rgb(255,255,255);
	background-color: rgb(75,166,255);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(51, 59, 73);
}
QPushButton:pressed {	
	background-color: rgb(33, 37, 43);
}</string>
         </property>
         <property name="text">
          <string>importieren</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="Line" name="libraryLine">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>70</y>
           <width>511</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(27, 29, 35)</string>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QListView" name="imageListView">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>90</y>
           <width>511</width>
           <height>351</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>511</width>
           <height>311</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>99999</width>
           <height>99999</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="autoScrollMargin">
          <number>16</number>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="resizeMode">
          <enum>QListView::Adjust</enum>
         </property>
         <property name="viewMode">
          <enum>QListView::IconMode</enum>
         </property>
         <property name="uniformItemSizes">
          <bool>true</bool>
         </property>
        </widget>
        <zorder>importButton</zorder>
        <zorder>libraryLineEdit</zorder>
        <zorder>libraryLabel</zorder>
        <zorder>libraryLine</zorder>
        <zorder>imageListView</zorder>
       </widget>
       <widget class="QWidget" name="classicPage">
        <widget class="QLabel" name="mainImageLabel">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>20</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255,255,255);</string>
         </property>
         <property name="text">
          <string>Bild</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="mainImageLineEdit">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>20</y>
           <width>441</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {	
	border: none;
	color: rgb(255,255,255);
	background-color: rgb(27, 29, 35);
	border-radius: 5;
}</string>
         </property>
         <property name="text">
          <string>D:\Projekt\Uni\Medienverarbeitung\res\input.png</string>
         </property>
        </widget>
        <widget class="QPushButton" name="classicButton">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>50</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {	
	border: rgb(255,255,255);
	color: rgb(255,255,255);
	background-color: rgb(75,166,255);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(51, 59, 73);
}
QPushButton:pressed {	
	background-color: rgb(33, 37, 43);
}</string>
         </property>
         <property name="text">
          <string>erstellen</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="Line" name="classicLine">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>70</y>
           <width>511</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(27, 29, 35)</string>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="partialPage">
        <widget class="QProgressBar" name="importProgressBar">
         <property name="geometry">
          <rect>
           <x>500</x>
           <y>410</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QProgressBar {
	color: rgb(255,255,255);
     border-radius: 5px;
     background-color: rgb(27, 29, 35);
 }

 QProgressBar::chunk {
     border-radius: 5px;
     background-color: rgb(75,166,255);
 }</string>
         </property>
         <property name="value">
          <number>24</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
